book.toml

```diff
@@ -27,31 +27,11 @@ level = 0
 
 [output.linkcheck]
 command = "ci/linkcheck.sh"
-follow-web-links = true
-exclude = [
-    "crates\\.io",
-    "gcc\\.godbolt\\.org",
-    "youtube\\.com",
-    "youtu\\.be",
-    "dl\\.acm\\.org",
-    "cs\\.bgu\\.ac\\.il",
-    "www\\.amazon\\.com",
-    "www\\.rustaceans\\.org",
-    "play\\.rust-lang\\.org",
-    "tomlee\\.co",
-    "marketplace\\.visualstudio\\.com",
-    "objects\\.githubusercontent\\.com",
-    # The bug listing URL works only if an user is logged in, otherwise one gets 404.
-    "github\\.com/issues\\?q=.*",
-    # Similarly 500 is sometimes returned here.
-    "github\\.com/rust-lang/rust/pulls\\?q=.*",
-    # 401 is returned here for unknown reason
-    "github\\.com/wesleywiser/rustc-bootstrap-wpa-analysis",
-    # Handle: connection closed before message completed
-    "microsoft\\.com/en-us/research/publication/",
-    # 500 is returned for HEAD request
-    "code\\.visualstudio\\.com/docs/editor/tasks",
-]
```

Comment:
This long list of excludes to me suggests that it's not good in terms of signal-to-noise ratio, because we're already skipping the (arguably most import) github links (issues, PRs, commits, etc.).